src/blog/_posts/2018-11-06-opera-57.md

```diff
@@ -0,0 +1,151 @@
+now disabled by default on desktop platforms. This is to avoid confusion when
+feature detecting touch support - use `navigator.maxTouchPoints` for
+touchscreen detection and `window.TouchEvent` touch event detection instead. To
+support both touch and mouse input, consider using the [Pointer Events](https://w3c.github.io/pointerevents/) API.
+
+Event listeners for the associated touch events can still be added via
+`EventTarget.prototype.addEventListener`, like:
+
+    document.addEventListener("touchstart", ...);
+
+## Options dictionary for postMessage methods
+
+An optional [`PostMessageOptions` dictionary](https://html.spec.whatwg.org/multipage/web-messaging.html#postmessageoptions)
+can now be passed to the
+[`postMessage`](https://html.spec.whatwg.org/multipage/web-messaging.html#dom-window-postmessage-options)
+method on `DedicatedWorkerGlobalScope`, `Worker`, `ServiceWorker`, `ServiceWorker`, and `Window`.
+
+Thus what was previously written as:
+
+    window.postMessage('my message', 'https://origin.example.com');
+
+can now also be written as:
+
+    window.postMessage('my message', { targetOrigin: 'https://origin.example.com' });
+
+## Media and Real-time Communications
+
+* [RTCPeerConnection.prototype.getConfiguration](https://w3c.github.io/webrtc-pc/#dom-rtcpeerconnection-getconfiguration)
+  has been added to allow accessing the last configuration set by
+  [RTCPeerConnection.prototype.setConfiguration](https://w3c.github.io/webrtc-pc/#dom-rtcpeerconnection-setconfiguration).
+* The `Opus` codec is now supported in mp4 (ISO-BMFF) containers with Media Source Extensions (MSE).
+* The new SourceBuffer.changeType() method allows for improved cross-codec or cross-bytestream transitions
+  within a SourceBuffer. It would for example allow changing the type of media bytes append to a media stream.
+
+## Network
+
+### Intervention Reports
+
+An "intervention" happens when the user agent (the browser) willfully violates
+the request of a web application in order to improve performance, security or
+eliminate behavior that is considered annoying to a user. These interventions
+can now be reported either by using the
+[`Report-To` HTTP Response header](https://w3c.github.io/reporting/#header) or the
+[`ReportingObserver` API](https://w3c.github.io/reporting/#observers).
+
+### TLS 1.3
+
+[TLS 1.3](https://tools.ietf.org/html/rfc8446) is a new version of the TLS
+protocol that sports a simpler, less error-prone design which improves both
+efficiency and security. This is achieved by, among other things, reducing the
+number of round-trips needed to setup a connection, and remove a number of
+insecure legacy options.
+
+## Layout / CSS
+
+* The `<rp>` element is now `display: none` by default even outside a `<ruby>` element.
+* The behavior of percentage row tracks and gutters in grid containers with
+indefinite heights has been updated, and now behaves the same as (grid) columns.
+
+##  Miscellanea
```

Comment:
Isn't misc already "plural", so this is many mixes of information?

Reply:
"A collection of diverse items" (Misc is adverb? So would need to be "Miscellaneous items" or so I think.)